release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -772,6 +772,15 @@ If you customized the default OpenSSH `/etc/ssh/sshd_config` server configuratio
 
 The {cert-manager-operator} 1.11 is now generally available in {product-title} 4.14 as well as {product-title} 4.13 and {product-title} 4.12.
 
+[discrete]
+[id="ocp-4-14-ovn-k-interconnect"]
+=== OVN-Kuberenetes interconnect
+{product-title} {product-version} introduces OVN-Kubernetes interconnect architecture to {openshift-networking}. This techincal change enables scaling clusters linearly with the number of nodes with a perfomance that is similar to OpenShift SDN.
```

Comment:
_scaling clusters_ almost sounds like a type of cluster, to me at least. I, personally, don't like the two _with_'s so close together. 
```suggestion
{product-title} {product-version} introduces OVN-Kubernetes interconnect architecture to {openshift-networking}. This technical change enables you to scale clusters linearly based on the number of nodes with a perfomance that is similar to OpenShift SDN.
```